Proposed/asc049.md

```diff
@@ -2,25 +2,35 @@
 Add `concentration` Unit to `Units` Definitions
 
 #### **Authorship**
-Alex Hadik alex@transcriptic.com
+Alex Hadik alex.hadik@strateos.com
 
 #### **Motivation**
 We should formalize how we represent concentration so that we have a standard way of defining solution concentrations.
 
 #### **Proposal**
-Add an arbitrary `item` "unit" to the `units` definition list. This is used as a catch all for discrete items such as cells, beads, etc.
 
-Add three types of concentration units to the `units` definition list: mass, molarity and arbitrary item. These concentration types will be defined as follows:
+##### Mass, Volume and Moles
+Add three types of concentration units to the `units` definition list: mass, volume and molarity. These concentration types will be defined as follows:
 
 - MassConcentration: `milligram/milliliter`, `nanogram/microliter`
-- MolarConcentration: `mole/liter`, `millimole/liter`, `micromole/liter`
-- ItemConcentration: `item/liter`, `item/milliliter`, `item/microliter`
+- VolumeConcentration: `milliliter/milliliter`, `microliter/microliter`
+- MoleConcentration: `mole/liter`, `millimole/liter`, `micromole/liter`
 
-These units are validated by validating the numerator and denominator units. The numerator must be any valid unit from the following set: (`Mass`, `Matter`, `Volume`, `Item`) The denominator must be a recognized unit from the `Volume` category.
+These units are validated by validating the numerator and denominator units. The numerator must be any valid unit from the following set: (`Mass`, `Matter`, `Volume`) The denominator must be a recognized unit from the `Volume` category.
 
 Given this validation approach, it should be noted that the proper format is `singular/singular` as in `milligram/milliliter` - NOT `milligrams/milliliter`.
 
 Put together, the format is `Float:Concentration`.
 
+##### Molar
+In addition to `Float:Concentration`, the standard Molar unit can be added as follows:
+
+| Unit       | Abbreviation | Multiplier |
+|------------|--------------|------------|
+| molar      | M            | 1          |
+| millimolar | mM           | 1e-3       |
+| micromolar | uM           | 1e-6       |
```

Comment:
Let's us the unicode `mu` here, i.e. μ

Reply:
That seems really hard to type and prone to error, am I correct in thinking that?

Reply:
It is true that it is harder to type, but it is by definition more correct. For example, looking at NIST (https://www.nist.gov/pml/weights-and-measures/metric-si-prefixes), μ is chosen.

Hmm we don't have a formal definition of the abbreviation definition and where/how that'll be used. I can see us using the abbreviation for rendering purposes, which in that case, the correct rendering of it would be much more important than the input difficulties.

If we're using it as an acceptable input, that will require more thought. Generally, `M` by itself denotes Mega, and publications use an italicized M or a small caps M as a workaround for derived compound units.